sycl/test/esimd/on-device/BitonicSortK.cpp

```diff
@@ -0,0 +1,718 @@
+// TODO enable on WIndows
+// REQUIRES: linux
+// REQUIRES: gpu
```

Comment:
```suggestion
// REQUIRES: linux, gpu
```
REQUIRES enables the test if all expressions are true.

Reply:
AFAIU, those two forms are equivalent, right? The 'REQUIRES: linux' should go away pretty soon, so I'd prefer just to remove the whole line rather then edit when this happens.